test/jdk/java/sql/testng/test/sql/TimestampTests.java

```diff
@@ -642,6 +642,35 @@ public void test52(long value, String ts) {
         assertEquals(ts1.toString(), ts, "ts1.toString() != ts");
     }
 
+    @Test
+    public void test53() {
+        // The latest Instant that can be converted to a Timestamp.
+        Instant instant1 = Instant.ofEpochSecond(Long.MAX_VALUE / 1000, 999_999_999);
+        assertEquals(instant1, Timestamp.from(instant1).toInstant());
+
```

Comment:
Not sure if you want to test the earliest Instant too, but it's `Instant.ofEpochSecond((Long.MAX_VALUE / -1000), 0)`

Reply:
I think the three overflow cases we have are probably enough? (Just too big, as big as possible, as small as possible.)